Testimonial (#64)
amplifying-fsharp/amplifying-fsharp.github.io: src/content/testimonials/RonaldSchlenker.md

Bring Back the Joy
---

The outcome of approaching a problem is usually so much more than just the solution you were looking for. While learning F#, I stumbled upon didactic masters such as Scott Wlaschin, which probably almost every F# developer knows. His legendary "F# For Fun and Profit", talks like "Four Languages From Forty Years Ago" or the book "Domain Modeling Made Functional" turbo-boosted my thinkung and dev-skills. There's a lot of money to make by selling trainings, scrum-here, agile-there, architecture-whatever. Buy it all - gladly also from me. And: Dig into Scott Wlaschin's stuff. It's a treasure chest filled with knowledge and rebooted my joy in programming.

> Taking every possible turn on my journey, I discovered people like Krzysztof Cieślak, who virtually single-handedly created the F# extension "Ionide" for VSCode from scratch. There's Tomas Petricek, a master mind who's doing tons of crazy-cool stuff, from academic papers to talks like "Write your own Excel in 100 lines of F#" or his hand-crafted F#-written Commodore C64 emulator. To me, this is just lovely, sympathic, and personal: There's someone trying to honestly share his knowledge and ideas, by adressing the good memories of your jounger self. Isn't that a masterpiece in communication? Who doesn't want to learn from such people?

Reviewer: ```suggestion
Taking every possible turn on my journey, I discovered people like Krzysztof Cieślak, who virtually single-handedly created the F# extension "Ionide" for VSCode from scratch. There's Tomas Petricek, a master mind who's doing tons of crazy-cool stuff, from academic papers to talks like "Write your own Excel in 100 lines of F#" or his hand-crafted F#-written Commodore C64 emulator. To me, this is just lovely, sympathetic, and personal: There's someone trying to honestly share his knowledge and ideas, by addressing the good memories of your younger self. Isn't that a masterpiece in communication? Who doesn't want to learn from such people?
```

The F# Community is rather small, but filled with those unique and highly-skilled personalities. More than that, I felt that they are all welcoming, friendly, approachable and willing to help. From my personal experience, most F# users care a lot about improving, developing and spreading F#. It's easy for anyone to get involved. There are so much possibilities from commenting, writing or implementing language features, contributing to libraries, writing own stuff and getting in touch with others: It's all easy and low-barrier. Instead of "strong opinions", I experienced open-minds and thoughtfulness. I have no doubt that talent and hard work can be keys to success, but I'm also convinced that you can get as far with curiosity and joy.

Old School Players to New School Fools
---

> A whole can be more than the sum of it's parts. That sounds trivial, and it means: Being part of something, knowing people and being bound to them in a good way, can release a lot of energy. It motivates, it brings joy and leads to new ideas and ways that weren't even seen before. Amplifying F# is an initiative founded by Edgar Gonzales, David Schäfer, Florian Verdonck, and Jimmy Byrd. They literally took the initiative, using all their talent, hard work, curiosity and joy to bring people together. It's already a success story, of which I'm proud to be now a part of.

Reviewer: ```suggestion
A whole can be more than the sum of it's parts. That sounds trivial, and it means: Being part of something, knowing people and being bound to them in a good way, can release a lot of energy. It motivates, it brings joy and leads to new ideas and ways that weren't even seen before. Amplifying F# is an initiative founded by Edgar Gonzales, David Schaefer, Florian Verdonck, and Jimmy Byrd. They literally took the initiative, using all their talent, hard work, curiosity and joy to bring people together. It's already a success story, of which I'm proud to be now a part of.
```

> Things change; this is an inevitable truth. I'm honestly happy for people who took a step and moved on. While this might create a vacuum or lead so sadness, there are alternatives we can choose from: Knowing that we have something great in our hands, we can turn sadness into happiness. Get things going, keep things rolling. Don't wait for others, and take initiative, get involved and be a part of something new. It's all on us.

Reviewer: ```suggestion
Things change; this is an inevitable truth. I'm honestly happy for people who took a step and moved on. While this might create a vacuum or lead to sadness, there are alternatives we can choose from: Knowing that we have something great in our hands, we can turn sadness into happiness. Get things going, keep things rolling. Don't wait for others, and take initiative, get involved and be a part of something new. It's all on us.
```
